.gitignore

```diff
@@ -22,3 +22,5 @@ Homestead.json
 composer.lock
 .phpunit.result.cache
 
+.idea/
```

Comment:
I think `.idea` files should be added to a user's global `.gitignore` file.

Reply:
Let me disagree with this :)

Reply:
I know...it's certainly subjective.  I used to toss quite a bit into `.gitignore` files, but after reading articles like [this](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/), I realized I could save myself and others a little time by moving a lot of stuff into my own global `.gitignore` file.

Anyway, I'm always open to (a reasonable amount of respectful) discussion/debate, so if this or anything else is sufficiently important to you, then please know that we can talk about it.  😃